client/server/render/index.js

```diff
@@ -125,6 +125,22 @@ export function render( element, key = JSON.stringify( element ), req ) {
 	} catch ( ex ) {
 		if ( process.env.NODE_ENV === 'development' ) {
 			throw ex;
+		} else {
+			try {
+				req.context.store.dispatch(
+					logToLogstash( {
+						feature: 'calypso_ssr',
+						message: 'Exception thrown on render',
+						extra: {
+							message: ex.message,
+							stack: ex.stack,
+						},
```

Comment:
Sending `user_id` (when user bootstrapping is on and the server actually knows who the user is) could be helpful for debugging, when we actually start solving the reported issues.